webapp-django/crashstats/signature/views.py

```diff
@@ -238,6 +238,58 @@ def signature_aggregation(request, aggregation):
 
 
 @waffle_switch('signature-report')
+@utils.json_view
+def signature_graphs(request, field):
+    '''Return the aggregation of a field. '''
+    params = get_validated_params(request)
+    if isinstance(params, http.HttpResponseBadRequest):
+        # There was an error in the form, let's return it.
+        return params
+
+    signature = params['signature'][0]
+
+    data = {}
+    data['aggregation'] = field
+
+    allowed_fields = get_allowed_fields(request.user)
+
+    # Make sure the field we want to aggregate on is allowed.
+    if field not in allowed_fields:
+        return http.HttpResponseBadRequest(
+            '<ul><li>'
+            'You are not allowed to aggregate on the "%s" field'
+            '</li></ul>' % field
+        )
+
+    current_query = request.GET.copy()
+    data['params'] = current_query.copy()
+
+    params['signature'] = '=' + signature
+    params['_results_number'] = 0
+    params['_results_offset'] = 0
+    params['_histogram.date'] = [field]
+
+    data['current_url'] = '%s?%s' % (
+        reverse('signature:signature_report'),
+        current_query.urlencode()
+    )
```

Comment:
Oops, this was naively copied over from the aggregations view. In fact it doesn't seem to be being used there either.